include/vcpkg/visualstudio.h

```diff
@@ -1,14 +1,28 @@
 #pragma once
 
+#include <vcpkg/vcpkgpaths.h>
+
+namespace vcpkg
+{
+    struct ToolsetsInformation
+    {
+        std::vector<Toolset> toolsets;
+
 #if defined(_WIN32)
```

Comment:
It avoids defining `get_localized_debug_info()` for other platforms and the whole system is of questionable value (on non-Windows) because it's _extremely_ VS-centric.

Reply:
Sure, I was just trying to optimize for the smallest amount of the product affected by macros, but isn't something I feel strongly about.